docs/user-guide/private-repositories.md

```diff
@@ -116,6 +116,42 @@ The Argo CD UI don't support configuring SSH credentials. The SSH credentials ca
 argocd repo add git@github.com:argoproj/argocd-example-apps.git --ssh-private-key-path ~/.ssh/id_rsa
 ```
 
+### GitHub App Credential
```

Comment:
@stlava Ah thanks - I somehow couldn't find this. Would you mind also updating the "declarative setup" documentation to provide an example of the repository config?

https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/declarative-setup.md